daemon/daemon_windows.go

```diff
@@ -340,7 +340,9 @@ func (daemon *Daemon) initNetworkController(config *Config, activeSandboxes map[
 
 		// If there is no nat network create one from the first NAT network
 		// encountered
-		if !defaultNetworkExists && runconfig.DefaultDaemonNetworkMode() == containertypes.NetworkMode(strings.ToLower(v.Type)) {
+		if !defaultNetworkExists &&
+			runconfig.DefaultDaemonNetworkMode() == containertypes.NetworkMode(strings.ToLower(v.Type)) &&
+			n == nil {
```

Comment:
The comment includes that it fixes https://github.com/docker/docker/issues/30102. Should I do a separate PR for that?